include/fluent-bit/flb_config.h

```diff
@@ -52,10 +52,22 @@
  * pointers.
  */
 
+#define FLB_CONTEXT_EV_SIGNAL      (1 << 0) /* 1 */
+
+enum ctx_signal_type {
```

Comment:
Please use a regular integer (probably `uint64_t` to ensure that we use the same unit size in every system)  constant instead of an enum for these.